sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -134,8 +132,11 @@ def _format_url(self, hostname: str) -> str:
 
     @classmethod
     def from_queue_url(
-        cls, queue_url: str,
-        credential: Optional[Union[str, Dict[str, str], "AzureNamedKeyCredential", "AzureSasCredential", "TokenCredential"]] = None,  # pylint: disable=line-too-long
+        cls,
+        queue_url: str,
+        credential: Optional[
+            Union[str, Dict[str, str], "AzureNamedKeyCredential", "AzureSasCredential", "TokenCredential"]
+        ] = None,  # pylint: disable=line-too-long
```

Comment:
I just noticed this here but is this line too long now? My guess is not so this Pylint ignore can be removed. This would apply to a ton of places throughout this review. Running pylint may catch this as bad ignore now?